portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,38 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Sehhaj!</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
+
+        // Initialise messages with three hard-coded values
+        ArrayList<String> messages = new ArrayList<String>();
```

Comment:
Use List<String> on the declaration side of the expression. Unless you're actually relying on some specific features that ArrayList gives you it's better to leave that to the implementation details (the new ArrayList<String> side of things).
(This tends to be more important in a class interface than a local variable but it's a good practice to get into)

Reply:
Done